### ISSUE TITLE
Unable to Build Bundle Project with Visual studio 2022 17.5.0 Preview 2.0

### ISSUE THREAD

**Issue by Manish082007** [firegiant/HeatWaveSupport#14]
### HeatWave Version

0.9.2

### Visual Studio Version

17.5.0 Preview 2.0

### Repro Steps

1. Installed 0.9.2 HeaveWave Visual Studio Extension.
2. Installed Visual Studio 2022 Preview Version. (17.5.0 Preview 2.0)
3. Just create a New Bundle Project & Add some MSI into chain element.
4. **Build Project & It will fail with below Error. I have tried many ways but the same Error. Tried to build with Wix command line too but same Error.**

WIX0200	The BootstrapperApplication element contains an unhandled extension element 'WixStandardBootstrapperApplication'. Please ensure that the extension for elements in the 'http://wixtoolset.org/schemas/v4/wxs/bal' namespace has been provided.

### Actual Result

Error While Building Bundle Project from Visual studio & `Wix build bundle.wxs` command line too.

WIX0200	The BootstrapperApplication element contains an unhandled extension element 'WixStandardBootstrapperApplication'. Please ensure that the extension for elements in the 'http://wixtoolset.org/schemas/v4/wxs/bal' namespace has been provided.

### Expected Result

It should Build the project and generate the .exe file.

**Comment by chrpai** [comment]
I'm unable to reproduce this.  The WiX v4 Bundle project template contains the correct refernce:

    <ProjectReference Include="..\ProductName1Setup\ProductName1Setup.wixproj" />

I add a project reference to an MSI project and the MsiPackage element to the chain and it builds fine.

Perhaps this is a problem with @Manish082007 's  nuget package sources.

**Comment by Manish082007** [comment]
@chrpai - No, I am adding MSI file reference directly. Problem is If i build it with Visual Studio 17.4.3 Version, it is building correctly. But if I Build same project in Visual Studio 17.5 Preview 2, It is not building.

**Comment by chrpai** [comment]
Sorry, I meant to paste this for the correct reference:

```
  <ItemGroup>
    <PackageReference Include="WixToolset.Bal.wixext" Version="4.0.0-rc.1" />
  </ItemGroup>

```

You need to check your nuget package source configuration in your 17.5 Preview 2 instance.  It's probably missing nuget.org.

**Comment by Manish082007** [comment]
@chrpai - I have the same 4.0 RC1 reference in the wixproj file of bundle project. I checked my nuget package sources are correct. 
I am using the same reference in the MSI Package project as well and I am able to build project & able to generate MSI too. 
If my nuget.org is missing then it should build for MSI package project.

Let me know, Any more details are required from my side.

**Comment by FireGiantHelp** [comment]
We can reproduce this internally in Visual Studio 17.5.0 Preview 2.0. This issue is on our backlog to fix.

**Comment by FireGiantHelp** [comment]
We expect this issue is fixed in HeatWave v0.9.3 now available in Visual Studio Marketplace so Visual Studio should provide notification of the update.

**Comment by Manish082007** [comment]
After Updating to 0.9.3, I am able to Build project successfully. Thank you so much for the quick help.
